### PR TITLE
Regenerate with latest gir / gir-files

### PR DESCRIPTION
**Pull request by sdroege**
@GuillaumeGomez Looks like this wasn't regenerated since a long time, completely outdated :)

### CODE REVIEW

**Review by GuillaumeGomez — APPROVED** [review]
Thanks! Merge once CI pass. :)

**Comment by bilelmoussaoui** [comment]
The builders were removed completely on this one since the migration to the new syntax on the gir file

**Comment by sdroege** [comment]
Indeed, I meant to investigate why they all disappeared. Thanks for checking :)

File: src/auto/gutter_renderer.rs [hunk 1]
```diff
@@ -34,105 +33,6 @@ glib_wrapper! {
     }
 }
 
-#[derive(Clone, Default)]
-pub struct GutterRendererBuilder {
```
Comment (sdroege) [inline src/auto/gutter_renderer.rs:38]: This is an abstract class and should've never had a builder :)

**Comment by sdroege** [comment]
Should be good to be merged now

**Comment by sdroege** [comment]
Fails for unrelated reasons, so let's get it in.

**Pull request merged**